### DISCUSSION TITLE
Building for fc41 doesn't work

### DISCUSSION THREAD

**Discussion by BurningPho3nix** [fedora-copr/copr#3400] [Q&A] [answered]
I tried multiple modifications to my rpm- build but non works for fc41.

Most recent:
https://copr.fedorainfracloud.org/coprs/burningpho3nix/Setup-Tool/build/7976354/

**Comment by FrostyX** [comment, marked as answer]
This sounds like you are missing a debug package 
https://docs.fedoraproject.org/en-US/packaging-guidelines/Debuginfo/

I think an easyfix for you could be disabling the debug package but putting this at the top of you specfile.

```
%global debug_package %{nil}
```

This isn't the correct solution but you could try if it helps, so you can narrow the scope of the issue.

I am moving this to a "discussions" because this is not a Copr bug.
